src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -800,6 +800,13 @@
         ],
         "Unit Localization": ["TemperatureUnit"],
         "Zone Management": ["TwoDCartesianMax"],
+        "TLS Certificate Management": [
+            "ClusterRevision",
+            "MaxRootCertificates",
+            "ProvisionedRootCertificates",
```

Comment:
Not a big deal, but list-typed attributes don't need to be listed here.  The system knows they have to be handled via AttributeAccessInterface or equivalent.

Reply:
Didn't know, thanks! (removed)